examples/federated-learning/func_lib.py

```diff
@@ -5,9 +5,11 @@
 
 from convert import decode
 
+from util import pin_to_owner
+
 ### Example model_fns ###
 
-def default_model_fn(data_owner):
+def _model_fn(data_owner):
```

Comment:
suggest calling this `_vanilla_model_fn` or `_keras_model_fn` or something to avoid clash with the base class's model_fn variable names

Reply:
`_base_model_fn`?